Extensions/Kayenta/Src/Tasks/createCanaryAnalysisRequestV0/task.json

```diff
@@ -0,0 +1,298 @@
+{
+  "id": "13a91ac7-d17e-4ca6-a4c0-723df247dd1b",
+  "name": "createCanaryAnalysisRequest",
+  "friendlyName": "Kayenta Canary Judge",
+  "description": "Invokes Kayenta Judge For Automated Canary Analysis",
+  "author": "Microsoft",
+  "helpMarkDown": "",
+  "category": "Utility",      
+  "visibility": [
+    "Release"
+  ],
+  "runsOn": [
+    "ServerGate"
+  ],
+  "version": {
+    "Major": 0,
+    "Minor": 154,     
+    "Patch": 2
+  },
+  "instanceNameFormat": "Kayenta Canary Judge",
+  "groups": [
+    {
+      "name": "accountConfiguration",
+      "displayName": "Account Configuration",
+      "isExpanded": true
+    },
+    {
+      "name": "thresholds",
+      "displayName": "Thresholds",
+      "isExpanded": true
+    },
+    {
+    "name": "executionRequest",
+    "displayName": "Analysis Configuration",
+    "isExpanded": false
+    },
+    {
+    "name": "scopes",
+    "displayName": "Scopes",
+    "isExpanded": false
+    }
+],
+  "inputs": [
+    {
+        "name": "KayentaConnection",
+        "type": "connectedService:kayenta",
+        "label": "Kayenta connection",
+        "defaultValue": "",
+        "required": "true",
+        "helpMarkDown": "Connection to the Kayenta instance used for canary analysis."
+    },
+    {
+      "name": "metricsAccountName",
+      "type": "string",       
+      "label": "Metric account",
+      "required": "true",
+      "helpMarkDown": "Name of metric account configured for Kayenta Ex : datadog-account",
+      "groupName": "accountConfiguration",
```

Comment:
Here as well. change "metric" to "metrics".
Also remove extra spacing between "Ex" and ":"